build_tools/buildkite/postsubmit.yml

```diff
@@ -0,0 +1,14 @@
+# Copyright 2022 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+steps:
+  - label: "Execute gcmn-test-pipeline"
+    agents:
+      queue: "orchestration"
+      security: "submitted"
+    commands: |
+      export BUILDKITE_ACCESS_TOKEN="$(gcloud secrets versions access latest --secret=iree-buildkite-presubmit-pipelines)"
+      ./build_tools/buildkite/wait_for_pipeline_success.py gcmn-test-pipeline
```

Comment:
Is Buildkite automatically setting `BUILDKITE_BRANCH` and the other env vars this script uses? Could add a comment explaining how each option is passed through. (either in the scripts or maybe in a README to explain how the scripts all work together)

Reply:
It is. Documentation incoming